tests/rptest/services/redpanda.py

```diff
@@ -2870,7 +2870,8 @@ def remove_local_data(self, node):
 
     def redpanda_pid(self, node):
         try:
-            cmd = "ps ax | grep -i 'redpanda' | grep -v grep | grep -v 'version'| grep -v \"\[redpanda\]\" | awk '{print $1}'"
+            cmd = "pgrep --exact redpanda"
```

Comment:
nice. i was going to suggest that we use the pid file that redpanda writes to disk, but this looks promising!